### ISSUE TITLE
How to prevent UnusedVariable with variable used in an include() or require()

### ISSUE THREAD

**Issue by jozydapozy** [sirbrillig/phpcs-variable-analysis#64]
I've this method in a class:
```
public function render() {
    global $app
    require($this->getFilename());
}
```
This raises this warning:

> Unused global variable $app. (VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable) 

I'm using the `$app` in the included file though. How should I fix this without using an (in this case unwanted) ignore-regexp?

**Comment by kkmuffme** [comment]
You can't. Atm this is the only solution:
// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable

**Comment by jozydapozy** [comment]
Ok, I think I can live with that ;) Thanks!
